CONTRIBUTING.md

```diff
@@ -1,8 +1,8 @@
 # Contributing Guidelines
-* You can join our [slack channel](http://systers.io/slack-systers-opensource/).  Each active repo has its own channel to direct questions to (for example #powerup or #portal).  
+* You can join our [ AnitaB.org Open Source Zulip Channel](https://anitab-org.zulipchat.com/).  Each active repo has its own channel to direct questions to (for example [#powerup](https://anitab-org.zulipchat.com/#narrow/stream/222537-powerup) or [#portal](https://anitab-org.zulipchat.com/#narrow/stream/222540-portal)).  
```

Comment:
Can you please add here somewhere the specific stream dedicated to #vms?

Reply:
Okay